src/Microsoft.Identity.Web.OWIN/AppBuilderExtension.cs

```diff
@@ -175,6 +175,23 @@ public static IAppBuilder AddMicrosoftIdentityWebApp(
 
                                 if (clientInfoFromServer != null && clientInfoFromServer.UniqueTenantIdentifier != null && clientInfoFromServer.UniqueObjectIdentifier != null)
                                 {
+                                    var uniqueTenantIdentifierClaim = context.AuthenticationTicket.Identity.FindFirst(c => c.Type == ClaimConstants.UniqueTenantIdentifier);
+                                    var uniqueObjectIdentifierClaim = context.AuthenticationTicket.Identity.FindFirst(c => c.Type == ClaimConstants.UniqueObjectIdentifier);
```

Comment:
You can move this after the below block since it won't be used if the exception is thrown.